tools/osx/crosstool/BUILD.toolchains

```diff
@@ -52,6 +52,10 @@ OSX_TOOLS_CONSTRAINTS = {
         "@platforms//os:ios",
         "@platforms//cpu:x86_64",
     ],
+    "watchos_arm64": [
+        "@platforms//os:ios",
+        "@platforms//cpu:aarch64",
+    ],
```

Comment:
not sure if this will conflict with the one below or if there's a different CPU that should be used for one or the other?

Reply:
It looks like it may in the future. If the constraints are the same, toolchain resolution would pick this one first.
But, what is up with the name  watchos_arm64_*32*? 
That entry is strange with a name that implies 32 bits, while using aarch64.

Reply:
> arm64_32 is a variant of arm64 with 32-bit pointer sizes, used on Apple Watch Series 4 and later.

Reply:
I wasn't sure about these. But `ios_sim_arm64` and `ios_arm64` are identical here too, so if it's wrong what's in here already is wrong.